docs/topics/dokka-plugins.md

```diff
@@ -139,76 +126,45 @@ If you are using [JSON configuration](dokka-cli.md#run-with-json-configuration),
 the documentation of the plugins you are using. 
 
-Let's have a look at how you can configure the `DokkaBase` plugin, which is responsible for generating [HTML](dokka-html.md)
-documentation, by adding a custom image to the assets (`customAssets` option), by adding custom style sheets
-(`customStyleSheets` option), and by modifying the footer message (`footerMessage` option):
+Let's have a look at how you can configure the built-in HTML plugin by adding a custom image to the assets 
+(`customAssets` option), 
+custom style sheets (`customStyleSheets` option), and a modified footer message (`footerMessage` option):
 
 <tabs group="build-script">
-<tab title="Kotlin" group-key="kotlin">
+<tab title="Gradle Kotlin DSL" group-key="kotlin">
 
-Gradle's Kotlin DSL allows for type-safe plugin configuration. This is achievable by adding the plugin's artifact to 
-the classpath dependencies in the `buildscript` block, and then importing plugin and configuration classes:
+To configure Dokka plugins in a type-safe way, use the `dokka.pluginsConfiguration {}` block:
 
 ```kotlin
-import org.jetbrains.dokka.base.DokkaBase
-import org.jetbrains.dokka.gradle.DokkaTask
-import org.jetbrains.dokka.base.DokkaBaseConfiguration
-
-buildscript {
-    dependencies {
-        classpath("org.jetbrains.dokka:dokka-base:%dokkaVersion%")
-    }
-}
-
-tasks.withType<DokkaTask>().configureEach {
-    pluginConfiguration<DokkaBase, DokkaBaseConfiguration> {
-        customAssets = listOf(file("my-image.png"))
-        customStyleSheets = listOf(file("my-styles.css"))
-        footerMessage = "(c) 2022 MyOrg"
+dokka {
+    pluginsConfiguration.html {
+        customAssets.from("logo.png")
+        customStyleSheets.from("styles.css")
+        footerMessage.set("(c) Your Company")
     }
 }
 ```
 
-Alternatively, plugins can be configured via JSON. With this method, no additional dependencies are needed.
-
-```kotlin
-import org.jetbrains.dokka.gradle.DokkaTask
+For an example of Dokka plugins configuration, see the
+[Dokka's versioning plugin](https://github.com/Kotlin/dokka/tree/master/examples/gradle-v2/versioning-multimodule-example).
 
-tasks.withType<DokkaTask>().configureEach {
-    val dokkaBaseConfiguration = """
-    {
-      "customAssets": ["${file("assets/my-image.png")}"],
-      "customStyleSheets": ["${file("assets/my-styles.css")}"],
-      "footerMessage": "(c) 2022 MyOrg"
-    }
-    """
-    pluginsMapConfiguration.set(
-        mapOf(
-            // fully qualified plugin name to json configuration
-            "org.jetbrains.dokka.base.DokkaBase" to dokkaBaseConfiguration
-        )
-    )
-}
-```
+Dokka allows you 
+to extend its functionality 
+by [configuring custom plugins](https://github.com/Kotlin/dokka/blob/v2.1.0/examples/gradle-v2/custom-dokka-plugin-example/demo-library/build.gradle.kts).
+Custom plugins enable additional processing or modifications to the documentation generation process.
```

Comment:
Not sure what is the difference between "additional processing" and "modifications to the generation process".

Maybe we can merge this into the previous sentence, like "Dokka allows you to extend its functionality and modify the documentation generation process by configuring custom plugins"?